### ISSUE TITLE
Pip install fails for version 0.1.4

### ISSUE THREAD

**Issue by process1183** [AtsushiSakai/PyRoombaAdapter#16]
Hello. Attempting to install PyRoombaAdapter 0.1.4 from pip results in an error:

```
josh@roomba:~$ sudo pip3 install pyroombaadapter
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Collecting pyroombaadapter
  Downloading https://files.pythonhosted.org/packages/2a/a6/beb4c900dfaa642c04364e1de1a1262b02bd7611b7d7a67f4cd4a57db230/pyroombaadapter-0.1.4.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-d7kk3yw9/pyroombaadapter/setup.py", line 25, in <module>
        with open(PROJECT_PATH + "/VERSION", 'r') as fd:
    FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-install-d7kk3yw9/pyroombaadapter/VERSION'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-d7kk3yw9/pyroombaadapter/
```

**Comment by AtsushiSakai** [comment]
I released v0.1.5, could you try to install?

**Comment by process1183** [comment]
v0.1.5 installs successfully! Thanks!

```
josh@roomba:~$ sudo pip3 install pyroombaadapter
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Collecting pyroombaadapter
  Downloading https://www.piwheels.org/simple/pyroombaadapter/pyroombaadapter-0.1.5-py3-none-any.whl
Requirement already satisfied: pyserial in /usr/lib/python3/dist-packages (from pyroombaadapter) (3.4)
Installing collected packages: pyroombaadapter
Successfully installed pyroombaadapter-0.1.5
```
